### PR TITLE
ITEM.tsv translation

### PR DESCRIPTION
**Pull request by JayShee**
Item description WIP

### CODE REVIEW

File: ITEM.tsv [hunk 1]
```diff
@@ -8969,254 +8969,254 @@ ITEM_20151224_008968	Collection: Manahas
...
-ITEM_20151224_009005
-ITEM_20151224_009006
-ITEM_20151224_009007
-ITEM_20151224_009008
-ITEM_20151224_009009
-ITEM_20151224_009010
-ITEM_20151224_009011
-ITEM_20151224_009012
+ITEM_20151224_008972	Prison Administrator Prison Cutter's Cube
+ITEM_20151224_008973	Hungry Velia Monkey's Cube
```
Comment (Sunaries) [inline ITEM.tsv]: Velnia

File: ITEM.tsv [hunk 2]
```diff
@@ -8969,254 +8969,254 @@ ITEM_20151224_008968	Collection: Manahas
...
+ITEM_20151224_008974	Forest Protector Ferret Marauder's Cube
+ITEM_20151224_008975	Helgasercle's Cube
```
Comment (Sunaries) [inline ITEM.tsv]: Helgasercle Cube

File: ITEM.tsv [hunk 3]
```diff
@@ -8969,254 +8969,254 @@ ITEM_20151224_008968	Collection: Manahas
...
+ITEM_20151224_008976	Mirtis' Cube
```
Comment (Sunaries) [inline ITEM.tsv]: Mirtis Cube

File: ITEM.tsv [hunk 4]
```diff
@@ -8969,254 +8969,254 @@ ITEM_20151224_008968	Collection: Manahas
...
+ITEM_20151224_008977	Rexipher's Cube
```
Comment (Sunaries) [inline ITEM.tsv]: Rexipher Cube

File: ITEM.tsv [hunk 5]
```diff
@@ -8969,254 +8969,254 @@ ITEM_20151224_008968	Collection: Manahas
...
-ITEM_20151224_009006
-ITEM_20151224_009007
-ITEM_20151224_009008
-ITEM_20151224_009009
-ITEM_20151224_009010
-ITEM_20151224_009011
-ITEM_20151224_009012
+ITEM_20151224_008972	Prison Administrator Prison Cutter's Cube
+ITEM_20151224_008973	Hungry Velia Monkey's Cube
+ITEM_20151224_008974	Forest Protector Ferret Marauder's Cube
```
Comment (Sunaries) [inline ITEM.tsv]: Forest Protector Ferret Marauder Cube

**Comment by Sunaries** [comment]
@JayShee Could you change the following in SKILL:

SKILL_20151224_005872 -> Change "comrade" to "ally".

**Comment by JayShee** [comment]
No idea why there's 5000+ changed lines, but all the new translations are there.

**Pull request closed without merging**